data/newSeeds/000_dropData.js

```diff
@@ -0,0 +1,57 @@
+exports.seed = async function (knex) {
+    // Truncate instead of delete all tables to wipe PKs
+    await knex.raw(
+      'ALTER SEQUENCE service_entry_recipients_id_seq RESTART WITH 1; TRUNCATE service_entry_recipients CASCADE'
+    );
+    await knex.raw(
+      'ALTER SEQUENCE service_entry_providers_id_seq RESTART WITH 1; TRUNCATE service_entry_providers CASCADE'
+    );
+    await knex.raw(
+      'ALTER SEQUENCE service_entries_id_seq RESTART WITH 1; TRUNCATE service_entries CASCADE'
```

Comment:
I wonder how it will work here and with the other primary keys that are actually UUIDs.

I'm interested in seeing if this still works or if there will be a mismatch issue?

In the drop of the old migrations (where they had a string as primary in their first table), it looks like they only included the `TRUNCATE `part without the `ALTER SEQUENCE`.

Reply:
Would it be helpful, just to get this pushed, if I empty the dropData file, and we write it together?